src/middleware/thunk.js

```diff
@@ -1,5 +1,5 @@
 export default function thunkMiddleware({ dispatch, getState }) {
-  return (next) => (action) =>
+  return next => action =>
     typeof action === 'function' ?
       action(dispatch, getState) :
```

Comment:
Should we consider having the default thunk middleware send the same `methods` type object to `action` instead of splitting them into separate parameters like this? i.e.

```js
action({ dispatch, getState })
```